### PR TITLE
Fixes #4 | iwgetid instead of nmcli

### LINKED ISSUE #4: Doesn't work on non-english Ubuntu distros
Command `nmcli` is localized on my Ubuntu 14.04. Here is my output

```
$ nmcli -t -f active,ssid dev wifi
tak:'FRITZ!Box Fon WLAN 7360 SL'
nie:'HTC Portable Hotspot'
```

In this case regex `/^\s*yes:(.+)\s*$/gm` doesn't match. Using `iwgetid -r` might be an alternative solution.

### PR DESCRIPTION
**Pull request by msn0**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ module.exports = function (cb) {
 		cmd = '/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport';
 		args = ['-I'];
 	} else if (process.platform === 'linux') {
-		cmd = 'nmcli';
-		args = ['-t', '-f', 'active,ssid', 'dev', 'wifi'];
+		cmd = 'iwgetid';
+		args = ['-r'];
```
Comment (kevva) [inline index.js]: `--raw` instead of `-r`.

**Comment by kevva** [comment]
Could you squash the commits too? :)

**Comment by msn0** [comment]
ok :)

**Comment by kevva** [comment]
Thanks!

**Pull request merged**